packages/components/src/templates/next/components/complex/Hero/HeroFloating.tsx

```diff
@@ -61,7 +61,7 @@ export const HeroFloating = ({
         />
       </div>
       {/* Floating container */}
-      <div className="mx-6 -mt-[1.5rem] md:mx-10 md:-mt-[10rem] lg:mx-0 lg:-mt-[17.85rem]">
+      <div className="mx-6 -mt-[1.5rem] w-full max-w-[calc(100%-3rem)] md:mx-10 md:-mt-[10rem] md:max-w-[calc(100%-5rem)] lg:mx-0 lg:-mt-[17.85rem] lg:max-w-none">
```

Comment:
[question] why do we need to subtract the 3 rem and 5 rem for the `max-w`? went to look at the designs - seems like the empty white space is 6 units/10 units, which shuold be 1.5rem and 2.5 rem respectively

Reply:
oh cos we using `w-full` which excludes margin, effectively putting the margin outside the box and not being visible